### PR TITLE
Component review

### PR DESCRIPTION
**Pull request by monospaced**
Have left `Heading`, `Paragrpah` and `Markdown` for a follow up review, to keep things moving ⏩

### CODE REVIEW

File: puck.config.tsx [hunk 1]
```diff
@@ -427,10 +444,30 @@ export const config: Config<Props, RootProps> = {
...
+        description = "description",
+        headingLevel,
+        strapline = "strapline",
+      }) => (
         <Section>
-          <Hero description={description} strapline={strapline} />
+          <Hero
+            description={description}
+            headingLevel={headingLevel}
+            strapline={strapline}
```
Comment (chrisvxd) [inline puck.config.tsx:469]: Thoughts on dropping the Hero lock-up in favour of compositional Header and Paragraph, which now have max width support?

I was also going to suggest we keep the CTA on desktop - I don't know if it matters if there's a dupe.

**Review by chrisvxd — COMMENTED** [review]
Great! Left a couple of questions.

File: puck.config.tsx (hunk 1 above)
Comment (monospaced) [reply puck.config.tsx:469]: I suggest let's wait for Dropzones. But I do think having a Hero component is not a bad thing, as you often want to tweak the Hero behaviour beyond what the underlying typography can do, especially responsively, just cause it's the Hero and therefore very visible and important. Also I'm looking at Heading and Paragrpah next, and may want to remove or tweak the `maxWidth` props there anyway possibly replace with an `isMeasured` prop! Agree there's potential to drop it, but suggest give it a bit of time. On the CTA I don't think we need to double it on desktop, but if we did, I think the styling would need to change to reduce the double button effect, and also make it more propotional with the strapline. But I don't think we really need it and cleaner without.

Comment (chrisvxd) [reply puck.config.tsx:469]: I like `isMeasured`!

Sounds good re CTA.

File: components/Header/index.tsx [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
-        <Menu links={links} />
-      </nav>
-    </div>
-  </header>
-);
+const Header = ({ links }: { links: { href: string; label: string }[] }) => {
+  let LogoElement: any = "div";
+  const pathname = usePathname();
+
+  if (pathname === "/" || pathname === "/edit") {
```
[the remark this answers is not in the document]
Comment (monospaced) [reply components/Header/index.tsx:12]: You don't want the logo/company name as H1 on every page, but I think it makes sense for our homepage. The one bit of SEO we could _really_ use is getting bumped up higher in searches for "Measured". Kim from LBA (currently lead) noted that we're effectively un-findable through search right now. I think a lot of sites _don't_ do this cause; it's hard to implement in template systems, or their brand/product name SEO is already fine, or they just don't care (outline broken!), or they have an actual H1 text in the homepage that makes more sense that the logo for them. Hierarchically I think "Measured" sits above the other headings on the home page, and the strapline isn't really a heading (it's just big).

**Review by chrisvxd — APPROVED** [review]

**Pull request merged**